docs/tutorial/visualization.rst

```diff
@@ -0,0 +1,420 @@
+*************
+Visualization
+*************
+
+This tutorial is about visualization of COMPAS objects in Rhino, Grasshopper, and Blender.
+For information about standalone 2D visualizations (i.e. "plotting"), please visit the tutorial about :ref:`tutorial_plotters`.
+For information about standalone 3D visualizations, please visit the documentation of :py:mod:`compas_view2`: https://compas.dev/compas_view2.
+
+
+Object - Artist Pairs
+=====================
+
+Visualization of COMPAS objects is handled by artists.
+Each object type is paired with a corresponding artist type through a naming convention:
+``Box`` - ``BoxArtist``, ``Mesh`` - ``MeshArtist``, ``Robot`` - ``RobotArtist``.
+The individual artists are implemented for Rhino, Blender, and Grasshopper with a similar but not identical API,
+and have to be imported accordingly depending on the execution context.
```

Comment:
Actually, I really don't see the need of rolling back this anymore. I played with the current situation that uses the base `Artist` for everything, and it is incredibly effective and easy to use. I don't see